source/includes/get-started/quickstart.c

```diff
@@ -0,0 +1,41 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int main (int argc, char *argv[])
+{
+    mongoc_client_t *client;
+    mongoc_collection_t *collection;
+    mongoc_cursor_t *results;
+    const bson_t *doc;
+    char *str;
+
+    mongoc_init ();
+
+    client = mongoc_client_new ("<connection string>");
```

Comment:
Consider declaring variables when used to shorten example.

```suggestion
    mongoc_client_t * client = mongoc_client_new ("<connection string>");
```

C89 requires variable declarations to be at the top of scope, but C99 does not. The C driver no longer supports C89.